### ISSUE TITLE
Convention de nommage

### ISSUE THREAD

**Issue by streino** [ecolabdata/ecospheres-dashboard-backend#8]
Un peu paumé dans la convention de nommage. Je lis ici "score de `avg_quality`" au lieu "moyenne de quality->score".

Au minimum, `avg__quality__score` ?

_Originally posted by @streino in https://github.com/ecolabdata/ecospheres-dashboard-backend/pull/7#discussion_r1805050980_

**Comment by abulte** [comment]
J'aime quand même bien l'idée d'avoir l'info qu'on a été chercher la valeur dans un JSON nested.

Claude propose `nom_de_champ__sous_champ__calcul`.

**Comment by streino** [comment]
> J'aime quand même bien l'idée d'avoir l'info qu'on a été chercher la valeur dans un JSON nested.
> 
> Claude propose `nom_de_champ__sous_champ__calcul`.

Il a pas osé le triple underscore ? :sweat_smile: 

Ça me pose pas particulièrement de problème, mais en quoi ça serait utile de savoir qu'on l'a récupéré dans un champ json plutôt qu'un autre champ ? Savoir où chercher dans le modèle data.gouv (mouaif, y a pas tant de nesting) ? Autre chose ?

**Comment by abulte** [comment]
Yes faire le lien avec le modèle data.gouv.fr sans avoir à trop réfléchir. C'est peut-être pas indispensable mais je trouve ça confortable.

**Comment by DanFluture275** [comment]
Changements à reporter sur le dashboard également
